src/@types/State.ts

```diff
@@ -123,6 +123,7 @@ interface State {
    * Passed to Yjs and cleared on every action.
    * See: /redux-enhancers/pushQueue.ts.
    */
+  noteOffset: number | null
```

Comment:
Let's add a descriptive JSDOC comment here.

This accidentally picked up the comment for `pushQueue`.